src/edge/edgeManager.ts

```diff
@@ -54,14 +52,14 @@ export class EdgeManager {
         mapObj.set(Constants.moduleImagePlaceholder, imageName);
         mapObj.set(Constants.moduleFolderPlaceholder, moduleName);
         const deploymentGenerated: string = Utility.replaceAll(data, mapObj);
-        await fse.writeFile(targetDeployment, deploymentGenerated, {encoding: "utf8"});
+        await fse.writeFile(targetDeployment, deploymentGenerated, { encoding: "utf8" });

```

Comment:
Sorry for this kind of distractive change. This is the side-effect of a "format code"